spec/view_component/storybook/stories_spec.rb

```diff
@@ -33,7 +33,7 @@
               content: "Hello World!"
             },
             argTypes: {
-              content: { control: { type: :text }, name: "Content" }
+              content: { control: { type: :text }, name: "Content", description: nil }
```

Comment:
I don't see any examples in this spec file where we're setting a description and seeing it reflected in the csf json structure. It's critical that we cover the logic here. Can you add some specs?

Reply:
I've added some tests to check the CSF JSON and params values — let me know if it could so with some more examples, though 🙂